Makefile

```diff
@@ -56,49 +56,41 @@ run-dev-interactive run-bash: | build-dev
 run-cleanup: | build-dev
 	docker run $(dev_args) sh cleanup.sh
 
-run-black: | build-dev
-	docker run $(dev_args) black openslides_backend/ tests/
-
-run-isort: | build-dev
-	docker run $(dev_args) isort --recursive openslides_backend/ tests/
-
-run-flake8: | build-dev
-	docker run $(dev_args) flake8 openslides_backend/ tests/
-
-run-mypy: | build-dev
-	docker run $(dev_args) mypy openslides_backend/ tests/
-
```

Comment:
Please add back, since these are needed for code checking. `run-cleanup` does not check but fix instead.